web/styles/app_components.css

```diff
@@ -1009,3 +1009,18 @@ div.overlay {
         }
     }
 }
+
+.dropdown-widget-button {
+    padding: 4px 6px;
+    border: 1px solid hsl(0deg 0% 80%);
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    outline: none;
+
+    .fa-chevron-down {
+        position: relative;
+        top: -1px;
+    }
+}
```

Comment:
Just a small note that when rebasing to combine this with the background color PR, I needed to add this:

 +    background-color: hsl(0deg 0% 100%);

Reply:
Added.